### DISCUSSION TITLE
Issue ideas - Milestone 1

### DISCUSSION THREAD

**Discussion by ksvedal** [ksvedal/uu-chrome-extension#32] [Ideas]
Heres issue ideas

**Comment by andreascnits** [comment]
Fjern any fra content_script
ser ut som at message kan fint bare være en string, samme med response. Evt. et eget message / response objekt om vi finner ut at vi vil sende større ting frem og tilbake. Som en rapport

Og sender er ikke i bruk(?)

Husk å også oppdatere popup.tsx til å samsvare med endringene i content_script der sendMessage brukes
